app/lib/three_scale/policies/specification.rb

```diff
@@ -6,18 +6,43 @@ class Specification
       def initialize(doc)
         @errors = ActiveModel::Errors.new(self)
         @doc = doc
-        @validator = JSONValidator.new(doc)
       end
 
       attr_reader :errors, :doc
 
-      JSON_SCHEMA = {'$ref' => 'http://apicast.io/policy-v1/schema#'}.freeze
+      SCHEMAS_PATH = 'app/lib/three_scale/policies/schemas/'
+      POLICY_SCHEMAS_FILENAMES = %w[apicast-policy-v1.1.schema.json apicast-policy-v1.schema.json].freeze
```

Comment:
Previously, we only used a single schema (which was by the way outdated).
I've decided to add support to both schemas, and the policy may use either one of the supported.